[2025-06-27T04:52:01.442+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:52:01.559+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T04:52:01.584+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T04:52:01.586+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:52:01.651+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-17 02:10:00+00:00
[2025-06-27T04:52:01.703+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3690) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:52:01.726+0000] {standard_task_runner.py:72} INFO - Started process 3737 to run task
[2025-06-27T04:52:01.761+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-17T02:10:00+00:00', '--job-id', '923', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmpuu0_jq5r']
[2025-06-27T04:52:01.819+0000] {standard_task_runner.py:105} INFO - Job 923: Subtask create_ga4_events_dataset
[2025-06-27T04:52:02.316+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:52:02.752+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-17T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-17T02:10:00+00:00'
[2025-06-27T04:52:02.771+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:52:02.808+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T04:52:02.836+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T04:52:02.859+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T04:52:02.860+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T04:52:02.870+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T04:52:04.849+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T04:52:05.579+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:52:05.625+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-17T02:10:00+00:00, execution_date=20250617T021000, start_date=20250627T045201, end_date=20250627T045205
[2025-06-27T04:52:06.060+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:52:06.687+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T04:52:06.702+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T04:52:06.757+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-06-27T05:36:21.845+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T05:36:21.876+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T05:36:21.899+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T05:36:21.901+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T05:36:21.927+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-17 02:10:00+00:00
[2025-06-27T05:36:21.958+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-17T02:10:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmpc7te1rag']
[2025-06-27T05:36:21.954+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=117) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T05:36:21.965+0000] {standard_task_runner.py:105} INFO - Job 33: Subtask create_ga4_events_dataset
[2025-06-27T05:36:21.965+0000] {standard_task_runner.py:72} INFO - Started process 176 to run task
[2025-06-27T05:36:22.060+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [running]> on host ba0a3a3d8846
[2025-06-27T05:36:22.172+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-17T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-17T02:10:00+00:00'
[2025-06-27T05:36:22.177+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T05:36:22.206+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T05:36:22.218+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T05:36:22.221+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T05:36:22.223+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T05:36:22.224+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T05:36:23.620+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T05:36:23.785+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T05:36:23.786+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-17T02:10:00+00:00, execution_date=20250617T021000, start_date=20250627T053621, end_date=20250627T053623
[2025-06-27T05:36:23.927+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T05:36:24.101+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T05:36:24.108+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T05:39:29.163+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T05:39:29.189+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T05:39:29.223+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T05:39:29.224+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T05:39:29.274+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-17 02:10:00+00:00
[2025-06-27T05:39:29.299+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=119) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T05:39:29.306+0000] {standard_task_runner.py:72} INFO - Started process 165 to run task
[2025-06-27T05:39:29.308+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-17T02:10:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmpg5rg_k8p']
[2025-06-27T05:39:29.316+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask create_ga4_events_dataset
[2025-06-27T05:39:29.417+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [running]> on host 616f50362410
[2025-06-27T05:39:29.553+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-17T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-17T02:10:00+00:00'
[2025-06-27T05:39:29.556+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T05:39:29.577+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T05:39:29.582+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T05:39:29.586+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T05:39:29.589+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T05:39:29.591+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T05:39:31.051+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T05:39:31.267+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T05:39:31.271+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-17T02:10:00+00:00, execution_date=20250617T021000, start_date=20250627T053929, end_date=20250627T053931
[2025-06-27T05:39:31.386+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T05:39:31.522+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T05:39:31.536+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T05:48:50.256+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T05:48:50.278+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T05:48:50.290+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [queued]>
[2025-06-27T05:48:50.291+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T05:48:50.310+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-17 02:10:00+00:00
[2025-06-27T05:48:50.322+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=261) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T05:48:50.329+0000] {standard_task_runner.py:72} INFO - Started process 297 to run task
[2025-06-27T05:48:50.327+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_dev', 'create_ga4_events_dataset', 'scheduled__2025-06-17T02:10:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt_dev.py', '--cfg-path', '/tmp/tmpaz7a633r']
[2025-06-27T05:48:50.334+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask create_ga4_events_dataset
[2025-06-27T05:48:50.425+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_dev.create_ga4_events_dataset scheduled__2025-06-17T02:10:00+00:00 [running]> on host 546ad4141aeb
[2025-06-27T05:48:50.533+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_dev' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-17T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-17T02:10:00+00:00'
[2025-06-27T05:48:50.540+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T05:48:50.561+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T05:48:50.567+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T05:48:50.568+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events_dev.
[2025-06-27T05:48:50.568+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T05:48:50.569+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events_dev in project: sixth-topic-349709 
[2025-06-27T05:48:52.073+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T05:48:52.231+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T05:48:52.237+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_dev, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-17T02:10:00+00:00, execution_date=20250617T021000, start_date=20250627T054850, end_date=20250627T054852
[2025-06-27T05:48:52.366+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T05:48:52.613+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T05:48:52.617+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
